src/vmm_mad/remotes/lib/vcenter_driver/vcenter_importer.rb

```diff
@@ -521,5 +523,21 @@ def self.import_clusters(con_ops, options)
 
         end
 
+        def self.register_hooks
+            if !ONE_LOCATION
+                hooks_path = '/var/lib/one/remotes/hooks/vcenter/templates'
+            else
+                hooks_path = ONE_LOCATION + '/remotes/hooks/vcenter/templates'
+            end
```

Comment:
this should be moved to the beginning of the file (as in the other driver files)